hbogolib/base.py

```diff
@@ -183,3 +191,18 @@ def router(self, arguments):
         elif mode == HbogoConstants.ACTION_REMOVE_MY_LIST:  # remove from my list
             self.start()
             self.handler.procContext(HbogoConstants.ACTION_REMOVE_MY_LIST, content_id)
+
+        elif mode == HbogoConstants.ACTION_LIST_BOOKMARK:
```

Comment:
I prefer not to add complexity and additional actions to implement this, not really necessary, also this is specific only to the EU handler not needed for other handlers so this should be contained in the EU handler.

The List action can be simply used for this, the url should get constructed in addCat and cat simply added like is done for My List.